egui_demo_lib/src/apps/demo/demo_app_windows.rs

```diff
@@ -27,6 +27,7 @@ impl Default for Demos {
             Box::new(super::sliders::Sliders::default()),
             Box::new(super::widget_gallery::WidgetGallery::default()),
             Box::new(super::window_options::WindowOptions::default()),
+            Box::new(super::context_menu::ContextMenus::default()),
```

Comment:
Move it to the top. The demos are lexicographically sorted. Also it is a nice demo! ;)